spec/jobs/update_working_days_since_submission_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe UpdateWorkingDaysSinceSubmissionJob, type: :job do
+  describe "#perform" do
+    subject(:perform) { described_class.new.perform }
+
+    let(:tuesday_today) { Date.new(2022, 10, 4) }
+    let!(:draft_application_form) { create(:application_form) }
+    let!(:monday_application_form) do
+      create(:application_form, :submitted, submitted_at: Date.new(2022, 10, 3))
+    end
+    let!(:friday_application_form) do
+      create(:application_form, :submitted, submitted_at: Date.new(2022, 9, 30))
+    end
+
+    before { travel_to(tuesday_today) { perform } }
```

Comment:
Would `around` be clearer here than `before`? (I didn't know you could even do this so I might not be 'down with the kids' here)

Reply:
With around it would be something like:

```rb
around do |example|
  travel_to(tuesday_today) { example.run }
end

before { perform }
```

Personally I think this actually be a little more unclear to someone who isn't familiar with Rspec, but I'm happy to change it, what do you think?

Reply:
I don't mind that much if you disagree. 👍